### PR TITLE
Switch to prime react sidebar

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: explore/src/main/scala/explore/HelpBody.scala [hunk 1]
```diff
@@ -71,9 +69,6 @@ object HelpBody:
       .useEffectOnMountBy { (props, _, state) =>
         load(props.url).flatMap(v => state.set(Pot.fromTry(v)).to[IO])
       }
-      .useGlobalHotkeysBy((_, helpCtx, _) =>
-        UseHotkeysProps(CloseHelp.value, helpCtx.displayedHelp.set(none))
```
Comment (toddburnside) [inline explore/src/main/scala/explore/HelpBody.scala:75]: Is the `CloseHelp` hotkey being used anywhere else? If not, can it be deleted?

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/HelpBody.scala (hunk 1 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/HelpBody.scala:75]: You're right. I'll delete it. prime handles `closeOnEsc` by itself

Comment (rpiaggio) [reply explore/src/main/scala/explore/HelpBody.scala:75]: You can do `state.setAsync` (requires `import crystal.react.implicits.*`).

**Pull request merged**